tests/unit/chambers_of_deputies/test_chamber_of_deputies_dataset.py

```diff
@@ -0,0 +1,203 @@
+    def test_fetch_chambers_of_deputies_datasets(self, mocked_urlretrieve):
+        path_to_2017_dataset_zip = os.path.join(self.fixtures_path, 'Ano-2017.zip')
+        path_to_dataset_format_html = os.path.join(self.fixtures_path, 'datasets-format.html')
+        copy(path_to_2017_dataset_zip, self.path)
+        copy(path_to_dataset_format_html, self.path)
+        expected_files = ['Ano-2017.zip', 'datasets-format.html']
+
+        retrieved_files = self.subject.fetch()
+
+        self.assertTrue(mocked_urlretrieve.called)
+        self.assertEqual(mocked_urlretrieve.call_count, len(expected_files))
+        for retrieved_file, expected_file in zip(retrieved_files, expected_files):
+            self.assertIn(expected_file, retrieved_file)
+
+    def test_translate_2017_dataset(self):
+        xz_path = os.path.join(self.path, 'reimbursements-2017.xz')
+        csv_path = os.path.join(self.fixtures_path, 'Ano-2017.csv')
+        copy(csv_path, self.path)
+
+        data_frame_2017_as_csv = self._read_csv(csv_path)
+        self._assert_that_the_columns_are_as_expected_before_translation(data_frame_2017_as_csv)
+
+        self.subject.translate()
+        reimbursements_2017 = self._read_xz(xz_path)
+
+        self._assert_that_the_columns_are_as_expected_after_translation(reimbursements_2017)
+
+
+    def test_clean_2017_reimbursements(self):
+        copy(os.path.join(self.fixtures_path, 'reimbursements-2017.xz'), self.path)
+        file_path = os.path.join(self.path, 'reimbursements.xz')
+
+        self.subject.clean()
+
+        assert(os.path.exists(file_path))
+
+        dataset = pd.read_csv(file_path, compression='xz')
+        all_subquotas = [
+            'Maintenance of office supporting parliamentary activity',
+            'Locomotion, meal and lodging',
+            'Fuels and lubricants',
+            'Consultancy, research and technical work',
+            'Publicity of parliamentary activity',
+            'Purchase of office supplies',
+            'Software purchase or renting; Postal services; Subscriptions',
+            'Security service provided by specialized company',
+            'Flight tickets',
+            'Telecommunication',
+            'Postal services',
+            'Publication subscriptions',
+            'Congressperson meal',
+            'Lodging, except for congressperson from Distrito Federal',
+            'Automotive vehicle renting or watercraft charter',
+            'Aircraft renting or charter of aircraft',
+            'Automotive vehicle renting or charter',
+            'Watercraft renting or charter',
+            'Taxi, toll and parking',
+            'Terrestrial, maritime and fluvial tickets',
+            'Participation in course, talk or similar event',
+            'Flight ticket issue'
+        ]
+
+        present_subquotas = pd.unique(dataset['subquota_description'])
+        for subquota in present_subquotas:
+            assert(subquota in all_subquotas)
```

Comment:
Minor thing here. This structure would allow better error messages (if any):

```python
        for subquota in present_subquotas:
            with self.subTest():
                self.assertIn(subquota, all_subquotas)
```

Reply:
Done in 8f4603361060154a59c961035699bcbeffc15145